tool_commands/import_sufhyd/turtleurbanclasses_tests.py

```diff
@@ -1918,8 +1917,8 @@ class Bug02713(object):  # (unittest.TestCase):
 *LEI   20           20.2\000      1   -4.87   -4.87  22.05 01 00     0.300        00                                        0 3.00\000\001\xFF
 """
 
-    ## skipping tests requires python 2.7
-    ## @unittest.skip("this test is not yet working, so it's not included")
+    # skipping tests requires python 2.7
```

Comment:
Ik zou deze twee commentaarregels weghalen, die worden toch niet gebruikt?

Erg oude code trouwens als er nog "werkt pas als we 2.7 hebben" in staat :)